microservices/authentication/src/aggregates.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import (
+    annotations,
+)
+
+from minos.common import (
+    Aggregate,
+)
+
+
+class User(Aggregate):
```

Comment:
I think we need to rename this aggregate to avoid collisions with the one defined at the `user` microservice and include a reference to it. 

Maybe something like this would be an option:
```python
class Credential(Aggregate):
    username: str
    password: str
    active: bool
    user: ModelRef[User]
```

Reply:
Done! You're right.